pages/common/archwiki-rs.md

```diff
@@ -0,0 +1,20 @@
+# archwiki-rs
+
+> A CLI tool to read pages from the ArchWiki.
```

Comment:
```suggestion
> Read, search, and download ArchWiki pages.
```
We generally avoid descriptions with CLI/command-line and prefer starting with a verb in the imperative mood, as document in contributing-guides/style-guide.md.

What about this?